.github/workflows/container-autoupdate.yml

```diff
@@ -0,0 +1,34 @@
+name: Build, test and push container
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  push-master-container:
+    name: Refresh Rawhide container
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout anaconda repository
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+
+      - name: Build and push container
+        uses: docker/build-push-action@v2
```

Comment:
This is a bit awkward. It's one step for two distinct actions (building and pushing a container), and we want to put the validation *in between*. This causes the big code duplication and also a race condition -- the second build may look different from the first one. Instead, I suggest to do the build manually with `make anaconda-ci-build` (which will also validate that our Makefile rule works as expected), then run tests (`make container-ci`), and only *afterwards* push the container. That's a simple `docker tag` and `docker push`, so honestly I don't see the need to invoke an external action here.

Reply:
I'll try to experiment with this a bit more.

Reply:
I stand by my opinion that replacing build-push-action@v2 with a simple `docker tag && docker push` run command after `make anaconda-ci-build && make container-ci` is both simpler, structurally what we want, and much easier to reproduce locally.